msbuild/Xamarin.MacDev.Tasks.Core/Tasks/XcodeCompilerToolTask.cs

```diff
@@ -191,7 +191,8 @@ protected int Compile (ITaskItem[] items, string output, ITaskItem manifest)
 						var plist = PDictionary.FromFile (manifest.ItemSpec);
 
 						LogWarningsAndErrors (plist, items[0]);
-					} catch {
+					} catch (Exception ex) {
+						Log.LogError ("Failed to load {0} log file `{1}`: {2}", ToolName, manifest.ItemSpec, ex.Message);
```

Comment:
Won't this show an error for something that was ignored before?

Reply:
That was a good question though because I had to go back and check the code :)